### PR TITLE
upgrade edition nft and AllDaySeasonal nft

### PR DESCRIPTION
**Pull request by Deewai**

### CODE REVIEW

File: edition-nft/contracts/AllDaySeasonal.cdc [hunk 1]
```diff
@@ -59,17 +63,17 @@ pub contract AllDaySeasonal: NonFungibleToken {
     // totalSupply
     // The total number of NFTs that in circulation.
     //
-    pub var totalSupply:        UInt64
+    access(all) var totalSupply:        UInt64
```
Comment (loic1) [inline edition-nft/contracts/AllDaySeasonal.cdc:66]: Should we remove this field now that burning is not decrementing it anymore?

File: edition-nft/contracts/EditionNFT.cdc [hunk 2]
```diff
@@ -59,14 +61,14 @@ pub contract EditionNFT: NonFungibleToken {
     // totalSupply
     // The total number of {{ contractName }} NFTs that have been minted.
     //
-    pub var totalSupply:        UInt64
+    access(all) var totalSupply:        UInt64
```
Comment (loic1) [inline edition-nft/contracts/EditionNFT.cdc:64]: Same question

**Review by loic1 — APPROVED** [review]
nice! just have one question about `totalSupply`

File: edition-nft/contracts/AllDaySeasonal.cdc (hunk 1 above)
Comment (Deewai) [reply edition-nft/contracts/AllDaySeasonal.cdc:66]: good point

File: edition-nft/contracts/EditionNFT.cdc (hunk 2 above)
Comment (Deewai) [reply edition-nft/contracts/EditionNFT.cdc:64]: it is used as ID in editionNFT contract

File: edition-nft/contracts/AllDaySeasonal.cdc (hunk 1 above)
Comment (Deewai) [reply edition-nft/contracts/AllDaySeasonal.cdc:66]: deleted

**Pull request merged**